README.md

```
@@ -250,6 +250,69 @@ generated from workouts are accepted without issues.
+    # Session is expired. You'll need to log in again
+    garth.login("email", "password")
+    garth.save(".garth")
+
+with open("your_fit_file.fit", "rb") as fp:
+    response = garth.client.upload(fp, return_id=True)
+
+id_num = response["detailedImportResult"]["successes"][0]["internalId"]
+garth.client.rename(id_num, "A better title than 'Virtual Cycling'")
+```
+
+
```

Markdownlint (0.37.0)

Expected: 1; Actual: 2
Multiple consecutive blank lines

(MD012, no-multiple-blanks)